src/app/shared/footer.js

```diff
@@ -5,15 +5,37 @@ class Footer extends React.Component {
   render() {
     return (
 		<section className='menu'>
-			<div id='footer' className='content-container'>
+      <footer className='content-container clearfix'>
+
+			<div id='links-footer'>
 				<ul>
-					<li><a href="http://www.facebook.com/cwaboulder" target="out" title="Link to Conference on World Affairs Facebook Page">Facebook</a></li>
-					<li><a>Twitter</a></li>
-					<li><a>Youtube</a></li>
-					<li><a>Contact Us</a></li>
+          <li id="visible-menu"><a><i className="fa fa-bars"></i></a></li>
+          <div id="footer-links">
+  					<li><a href="http://www.facebook.com/cwaboulder" target="out" title="Link to Conference on World Affairs Facebook Page"><i className="fa fa-home"></i>Facebook</a></li>
```

Comment:
Nitpick: The alignment here needs reshaped pretty please. Hard to follow current alignment.